seed-job/buildtemplate.yaml

```diff
@@ -145,3 +148,19 @@ parameters:
   name: CCP_OPENSHIFT_SLAVE_IMAGE
   required: true
   value: registry.centos.org/pipeline-images/ccp-openshift-slave:latest
+- description: Amount of CPU to be requested from OpenShift to start seed-job slave pod
+  displayName: seed job CPU
+  name: SEED_JOB_CPU
+  required: true
+- description: Amount of memory to be requested from OpenShift to start seed-job slave pod
+  displayName: seed job memory
+  name: SEED_JOB_MEMORY
+  required: true
+- description: Amount of CPU to be requested from OpenShift to start master-job slave pod
+  displayName: master job CPU
+  name: MASTER_JOB_CPU
+  required: true
+- description: Amount of memory to be requested from OpenShift to start master-job slave pod
+  displayName: master job memory
+  name: MASTER_JOB_MEMORY
+  required: true
```

Comment:
lets add some default values as well, also we need to add this to README

Reply:
I've avoided setting up default value here because:
- it's highly dependent on the environment/infrastructure someone spins things up on
- also depends on the container-index they want to process
- most importantly, an admin might miss taking a look at these parameters and understanding their purpose

Do you think these are valid reasons for not setting a default value for the two variables?

Reply:
I believe default is set to `0` by jenkins. so anyway this works